packages/v4/src/content/design-guidelines/content/units-and-symbols.md

```diff
@@ -0,0 +1,102 @@
+---
+id: Units and symbols
+section: UX writing
+---
+
+This page focuses on how to write units and symbols in an interface. For guidance on writing and displaying numeric values like date, time, and currency, see our [Numerics](/ux-writing/numerics) resource.
+
+Use consistent formatting, terminology, and symbols to display units of measure in your UI. Units of measure and symbols often appear when describing quantitative values including:
+
+- Date and time
+- Data storage
+- Data bandwidth
+- Currency
+
+When providing quantitative values in your UI, use the International System of Units (SI units), so that your units of measure can be understood by global users. Use the following base units and symbols to describe different quantities:
+- Length: meter (m)
+- Mass: kilogram (kg)
+- Electric current: ampere (A)
+- Thermodynamic temperature: kelvin (K)
+- Amount of substance: mole (mol)
+- Luminous intensity: candela (cd)
+
+## Time
+
+Whenever possible, write the full name of each time unit.
+
+Example: Write "6 minutes" instead of "6 min."
+
+If space is limited, use symbols to communicate the same units of time in less space: 
+
+- Millisecond (ms)
+- Second (s)
+- Minute (min)
+- Hour (hr)
+
+Avoid using symbols for days, weeks, months, and years.
+
+## Data storage
+
+Computer storage and memory are typically measured in bytes. Bytes can be shown in **binary units** (gibibytes) or **decimal/metric units** (gigabytes).
+
+Use binary units to express amounts of digital information in alignment with most computing and open source software. Keeping consistent eliminates confusion.
+
+| **Binary unit** | **Symbol** | **Value**        |
+|-----------------|------------|------------------|
+| 1 kibibyte      | KiB        | 1024             |
+| 1 mebibyte      | MiB        | 1024<sup>2</sup> |
+| 1 gibibyte      | GiB        | 1024<sup>3</sup> |
+| 1 tebibyte      | TiB        | 1024<sup>4</sup> |
+
+| **Metric unit** | **Symbol** | **Value**        |
+|-----------------|------------|------------------|
+| 1 kilobyte     | KB       | 1000             |
+| 1 megabyte     | MB        | 1000<sup>2</sup> |
+| 1 gigabyte     | GB        | 1000<sup>3</sup> |
+| 1 terabyte     | TB       | 1000<sup>4</sup> |
+
+## Data bandwidth
+
+Bandwidth is the amount of data transferred from one point to another within a network in a specified amount of time.
+
+Data transfer is primarily measured in two ways:
+- **Megabits/second (Mbps)**: Specifies download and upload speeds on the internet. 
+- **Megabytes/second (MBps)**: Specifies the quantity of data or file size transferred over time.
```

Comment:
We can remove the periods after these 2 items since they don't form complete sentences.